src/lib.rs

```diff
@@ -1290,12 +1287,33 @@ fn parse_impl<R: Read>(reader: R, map_path: Option<&Path>) -> Result<Map, TiledE
     }
 }
 
+fn default_file_loader(map_path: Option<PathBuf>)->impl FnMut(&str)->Result<Vec<u8>, TiledError> {
+    move |source: &str| {
+        let tileset_path = map_path.as_ref()
```

Comment:
You can do this to avoid needing `to_owned` in the `default_file_loader(Some(path.to_owned()))`s below.
```suggestion
fn default_file_loader(map_path: Option<&Path>)->impl FnMut(&str)->Result<Vec<u8>, TiledError> + '_ {
    move |source: &str| {
        let tileset_path = map_path
```